engine/src/windowing/window.cpp

```diff
@@ -25,7 +25,7 @@ namespace gen
 
 		// Set window hints
 		glfwWindowHint(GLFW_CLIENT_API, GLFW_NO_API);
-		glfwWindowHint(GLFW_RESIZABLE, GLFW_FALSE); // TODO: Later add support for resizing of windows.
+		glfwWindowHint(GLFW_RESIZABLE, GLFW_TRUE); // TODO: Later add support for resizing of windows.
```

Comment:
Just letting you know that dealing with window resizing involves a complex set of constrained operations with the swapchain. (Of course it's crucial to support it, this is FYI to expect that rabbit hole incoming soon.)

Reply:
Oops, forgot to disable that for the PR! Had to on for testing the callback. :P

Reply:
Yeah I look forward for handling resizing! I've already looked a wee bit into it and it seems like its going to be an interesting problem to solve. ^^